pkg/util/utils.go

```diff
@@ -171,3 +176,20 @@ func IsDirectory(path string) (bool, error) {
 	}
 	return fileInfo.IsDir(), nil
 }
+
+func GetKubernetesVersion() (semver.Version, error) {
+	if VERSION.Compare(semver.MustParse("0.0.0")) != 0 {
+		return VERSION, nil
+	}
+	r, _ := regexp.Compile(".*\nServer Version: v")
+	versions, err := exec.Command("kubectl", "version", "--short").Output()
```

Comment:
We can probably do this by accessing the API directly using client-go instead of shelling out. This will have trouble on Windows. I'll try to find an example.